sgp4/tests.py

```diff
@@ -164,6 +192,17 @@ class NewSatelliteObjectTests(TestCase, SatelliteObjectTests):
     def build_satrec(self, line1, line2):
         return Satrec.twoline2rv(line1, line2)
 
+    def build_satrec_from_sgp4init(self, satnum, jdsatepoch, bstar, ndot, nddot,
```

Comment:
Nice, thanks for adding these helper methods! Are you planning to call them from a test? Or, will it be a subsequent pull request that adds some tests that use them?

Reply:
That was apparently my oversight, and not completely understanding the unittests.  Will need to add methods to call them, and working on a PR that works for both new and legacy methods.

Reply:
Roger! Here are the docs that I used in writing this test suite, in case they're helpful:

https://docs.python.org/3.8/library/unittest.html#basic-example

Reply:
Figured out the tests.  Notes in the upcoming commit.